apps/L1E/instagram/instagram-frontend/src/components/Sidebar.tsx

```
@@ -32,10 +46,25 @@ export function Sidebar() {
+<DropdownMenu >
+  <DropdownMenuTrigger asChild className=""> 
+    <Link 
+      href="#" 
+      className="flex items-center w-full p-3 hover:text-primary gap-4 rounded-lg hover:bg-gray-100 transition-colors"  
+    >
+      <PlusSquare className="h-[24px] w-[24px]" />
+      <span className='text-[16px] font-normal'>Create</span>
+    </Link>
+  </DropdownMenuTrigger>
+  <DropdownMenuContent className='w-[154px] h-[72px] rounded-md flex flex-col mt-2 items-start justify-start mr-4'>
+    <DropdownMenuItem   onClick={(e) =>{ e.preventDefault() ,handlePostClick()}} className='flex pt-[6px] pr-[8px] justify-between items-center self-stretch'>
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)